sd/etcd/registrar.go

```diff
@@ -19,9 +25,27 @@ type Registrar struct {
 type Service struct {
 	Key           string // unique key, e.g. "/service/foobar/1.2.3.4:8080"
 	Value         string // returned to subscribers, e.g. "http://1.2.3.4:8080"
+	TTL           *TTLOption
 	DeleteOptions *etcd.DeleteOptions
 }
 
+// TTLOption allow setting a key with a TTL, and regularly refreshes the lease with a goroutine
```

Comment:
This comment describes the mechanics of what happens, but not what benefit it provides. Can you amend the comment to tell the user in which circumstances they would want to use this option? Also, please wrap the comment at 80 columns, and use proper punctuation (i.e. add a period to the end of the sentence).